<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessRoomForm</class>
 <widget class="QWidget" name="ChessRoomForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>棋牌房间管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>棋牌房间管理</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addRoomButton">
       <property name="font">
        <font>
         <family>仿宋</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>添加房间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deletRoomButton">
       <property name="font">
        <font>
         <family>仿宋</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>删除房间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeFormulaButton">
       <property name="font">
        <font>
         <family>仿宋</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>修改计费公式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="font">
        <font>
         <family>仿宋</family>
        </font>
       </property>
       <property name="text">
        <string>更新房间信息</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QTextEdit" name="roomInfoText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="chessRoomList">
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="openButton">
           <property name="font">
            <font>
             <family>仿宋</family>
            </font>
           </property>
           <property name="text">
            <string>开台</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="scheduleButton">
           <property name="font">
            <font>
             <family>仿宋</family>
            </font>
           </property>
           <property name="text">
            <string>预定</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="suspendButton">
           <property name="font">
            <font>
             <family>仿宋</family>
            </font>
           </property>
           <property name="text">
            <string>挂起</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="commoditySaleButton">
           <property name="font">
            <font>
             <family>仿宋</family>
            </font>
           </property>
           <property name="text">
            <string>购买商品</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeRoomButton">
           <property name="font">
            <font>
             <family>仿宋</family>
            </font>
           </property>
           <property name="text">
            <string>结账</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
